code/modules/mob/living/carbon/alien/special/alien_embryo.dm

```diff
@@ -91,6 +91,7 @@
 		owner.overlays += overlay
 
 		spawn(6)
```

Comment:
There are some `spawn()` in this proc, you may want to consider refactoring it, to slowly make xenomorph code up-to-date?

Reply:
Disregard this, it'd need almost the entire file to be refactored, which I will look into later :>